src/pages/sectionMeetingCsv/sectionMeetingCsv.js

```diff
@@ -0,0 +1,32 @@
+const router = require('express').Router({ mergeParams: true })
+const { sqlLoader } = require('@prairielearn/prairielib')
+const { Parser } = require('json2csv')
+const dbDriver = require('../../dbDriver')
+const asyncErrorHandler = require('../../asyncErrorHandler')
+const checks = require('../../auth/checks')
+
+const sql = sqlLoader.loadSqlEquiv(__filename)
+const parser = new Parser()
+
+router.get(
```

Comment:
This doesn't need to be a separate file - just put this under `router.get('/csv', ...)` in `sectionMeeting.js`. That helps to keep related functionality in the same place.

Reply:
Fair. Kind of copied your model for `courseInstanceAnalytics`